gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/dsl/ProguardSettings.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.desktop.application.dsl
+
+import org.gradle.api.file.ConfigurableFileCollection
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Property
+import org.jetbrains.compose.desktop.application.internal.notNullProperty
+import org.jetbrains.compose.desktop.application.internal.nullableProperty
+import javax.inject.Inject
+
+abstract class ProguardSettings @Inject constructor(
+    objects: ObjectFactory,
+) {
+    val version: Property<String> = objects.notNullProperty("7.2.2")
```

Comment:
What means 7.2.2?
It's default version of proguard?

Reply:
Yes, I will add a constant instead